threshold_encryption.go

```diff
@@ -0,0 +1,5 @@
+package cleisthenes
+
+var enc = func() {}
```

Comment:
1. change `var enc = func() {}` into `type Enc func ( ... ) <return type>`
2. plz specify function params and return type